<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>508</width>
    <height>259</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>508</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Security Control System</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../lock 1.png</normaloff>../../lock 1.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="runButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>325</y>
     <width>449</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">);</string>
   </property>
   <property name="text">
    <string>Система контроля доступа</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>295</y>
     <width>449</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="documentButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>225</y>
     <width>240</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">);</string>
   </property>
   <property name="text">
    <string>Журнал действий</string>
   </property>
  </widget>
  <widget class="QPushButton" name="workerButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>225</y>
     <width>240</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(54, 255, 40);</string>
   </property>
   <property name="text">
    <string>Список сотрудников</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>471</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Информационное окно для основной навигации по системе. Просмотр таблицы всех сотрудников системы. А также журнал событий в виде отчета о посещаемости сотрудников</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>511</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../icons/background_logo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
